### PR TITLE
Digest #14

### PR DESCRIPTION
**Pull request by pjazdzewski1990**

### CODE REVIEW

File: _posts/13-01-2015-digest14.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- Scalac Weekly Digest
+---
+
+Hi folks! 
+
+{DESC}
+
+* \[Android\] [MVVM pattern for Android](https://robobinding.github.io/RoboBinding/)
+
+* \[Scala\] [Why Scala?](http://softwarecorner.wordpress.com/2014/12/31/why-scala/)
```
Comment (LukasGasior1) [inline _posts/13-01-2015-digest14.md]: :+1:

File: _posts/13-01-2015-digest14.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+* \[Working\] [Books Every Entrepreneur Should Read](http://www.businessinsider.com/best-books-for-entrepreneurs-2014-9)
```
Comment (LukasGasior1) [inline _posts/13-01-2015-digest14.md]: :+1:

File: _posts/13-01-2015-digest14.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+ 
+* \[Javascript\] [Introduction to Gulp.js](https://medium.com/@lukyvj/an-introduction-to-gulp-js-295ee489e2c9) 
+
+* \[Artificial Intelligence\] [AI Solves Poker](http://www.geek.com/news/ai-solves-texas-hold-em-poker-and-becomes-unbeatable-1613099/) 
+
+* \[Working\] [Quantify your code](http://blog.newrelic.com/2014/12/16/quantify-your-code/)
```
Comment (LukasGasior1) [inline _posts/13-01-2015-digest14.md]: nothing special :-1:

File: _posts/13-01-2015-digest14.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+* \[Working\] [Quantify your code](http://blog.newrelic.com/2014/12/16/quantify-your-code/) 
+
+* \[Scala\] [Debugging Scala compiler](http://yefremov.net/blog/debugging-scala-compiler-magic/)
+ 
+* \[Javascript\] [Angular Air](https://ng-air.github.io/)
+
+* \[Working\] [Eric Meijer vs Agile](http://www.theregister.co.uk/2015/01/08/erik_meijer_agile_is_a_cancer_we_have_to_eliminate_from_the_industry/)
+
+* \[Scala\] [Functional Systems @ Twitter](http://www.infoq.com/presentations/twitter-scal a)
```
Comment (LukasGasior1) [inline _posts/13-01-2015-digest14.md]: extra space in url

File: _posts/13-01-2015-digest14.md (hunk 3 above)
Comment (kastoestoramadus) [inline _posts/13-01-2015-digest14.md]: Not essential  :-1:

File: _posts/13-01-2015-digest14.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+* \[Working\] [Quantify your code](http://blog.newrelic.com/2014/12/16/quantify-your-code/) 
+
+* \[Scala\] [Debugging Scala compiler](http://yefremov.net/blog/debugging-scala-compiler-magic/)
+ 
+* \[Javascript\] [Angular Air](https://ng-air.github.io/)
+
+* \[Working\] [Eric Meijer vs Agile](http://www.theregister.co.uk/2015/01/08/erik_meijer_agile_is_a_cancer_we_have_to_eliminate_from_the_industry/)
+
+* \[Scala\] [Functional Systems @ Twitter](http://www.infoq.com/presentations/twitter-scala)
```
Comment (kastoestoramadus) [inline _posts/13-01-2015-digest14.md]: PL person = programming language person :+1:
In descripton please underline that this is long report of their main project, it is not essential scala propaganda.

File: _posts/13-01-2015-digest14.md (hunk 1 above)
Comment (jczuchnowski) [inline _posts/13-01-2015-digest14.md]: :+1:

File: _posts/13-01-2015-digest14.md (hunk 2 above)
Comment (jczuchnowski) [inline _posts/13-01-2015-digest14.md]: :+1:

File: _posts/13-01-2015-digest14.md (hunk 3 above)
Comment (jczuchnowski) [inline _posts/13-01-2015-digest14.md]: :+1:

File: _posts/13-01-2015-digest14.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+* \[Android\] [MVVM pattern for Android](https://robobinding.github.io/RoboBinding/)
+
+* \[Scala\] [Why Scala?](http://softwarecorner.wordpress.com/2014/12/31/why-scala/)
+
+* \[Working\] [Books Every Entrepreneur Should Read](http://www.businessinsider.com/best-books-for-entrepreneurs-2014-9)
+ 
+* \[Javascript\] [Introduction to Gulp.js](https://medium.com/@lukyvj/an-introduction-to-gulp-js-295ee489e2c9) 
+
+* \[Artificial Intelligence\] [AI Solves Poker](http://www.geek.com/news/ai-solves-texas-hold-em-poker-and-becomes-unbeatable-1613099/)
```
Comment (jczuchnowski) [inline _posts/13-01-2015-digest14.md]: :+1:

File: _posts/13-01-2015-digest14.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+* \[Working\] [Books Every Entrepreneur Should Read](http://www.businessinsider.com/best-books-for-entrepreneurs-2014-9)
+ 
+* \[Javascript\] [Introduction to Gulp.js](https://medium.com/@lukyvj/an-introduction-to-gulp-js-295ee489e2c9) 
+
+* \[Artificial Intelligence\] [AI Solves Poker](http://www.geek.com/news/ai-solves-texas-hold-em-poker-and-becomes-unbeatable-1613099/) 
+
+* \[Working\] [Quantify your code](http://blog.newrelic.com/2014/12/16/quantify-your-code/) 
+
+* \[Scala\] [Debugging Scala compiler](http://yefremov.net/blog/debugging-scala-compiler-magic/)
```
Comment (jczuchnowski) [inline _posts/13-01-2015-digest14.md]: :+1:

File: _posts/13-01-2015-digest14.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+{DESC}
+
+* \[Android\] [MVVM pattern for Android](https://robobinding.github.io/RoboBinding/)
+
+* \[Scala\] [Why Scala?](http://softwarecorner.wordpress.com/2014/12/31/why-scala/)
+
+* \[Working\] [Books Every Entrepreneur Should Read](http://www.businessinsider.com/best-books-for-entrepreneurs-2014-9)
+ 
+* \[Javascript\] [Introduction to Gulp.js](https://medium.com/@lukyvj/an-introduction-to-gulp-js-295ee489e2c9)
```
Comment (jczuchnowski) [inline _posts/13-01-2015-digest14.md]: :-1:

File: _posts/13-01-2015-digest14.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+ 
+* \[Javascript\] [Introduction to Gulp.js](https://medium.com/@lukyvj/an-introduction-to-gulp-js-295ee489e2c9) 
+
+* \[Artificial Intelligence\] [AI Solves Poker](http://www.geek.com/news/ai-solves-texas-hold-em-poker-and-becomes-unbeatable-1613099/) 
+
+* \[Working\] [Quantify your code](http://blog.newrelic.com/2014/12/16/quantify-your-code/) 
+
+* \[Scala\] [Debugging Scala compiler](http://yefremov.net/blog/debugging-scala-compiler-magic/)
+ 
+* \[Javascript\] [Angular Air](https://ng-air.github.io/)
```
Comment (jczuchnowski) [inline _posts/13-01-2015-digest14.md]: :+1:

File: _posts/13-01-2015-digest14.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+
+* \[Working\] [Eric Meijer vs Agile](http://www.theregister.co.uk/2015/01/08/erik_meijer_agile_is_a_cancer_we_have_to_eliminate_from_the_industry/)
```
Comment (jczuchnowski) [inline _posts/13-01-2015-digest14.md]: :+1:

File: _posts/13-01-2015-digest14.md (hunk 7 above)
Comment (kastoestoramadus) [inline _posts/13-01-2015-digest14.md]: :+1:

File: _posts/13-01-2015-digest14.md (hunk 10 above)
Comment (kastoestoramadus) [inline _posts/13-01-2015-digest14.md]: :+1:

File: _posts/13-01-2015-digest14.md (hunk 6 above)
Comment (kastoestoramadus) [inline _posts/13-01-2015-digest14.md]: :+1:

**Comment by note** [comment]
"they've choose Scala" should be: "they've chosen Scala"

**Pull request merged**